#!/usr/bin/env python
PACKAGE = "tesseract_monitoring"
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

gen.add("publish_environment", bool_t, 1, "Set to True to publish environment", False)
gen.add("publish_environment_hz", double_t, 2, "Set the maximum frequency at which environment updates are published", 4, 0.1, 100.0)
gen.add("publish_geometry_updates", bool_t, 3, "Set to True to publish geometry updates of the environment", True)
gen.add("publish_state_updates", bool_t, 4, "Set to True to publish geometry updates of the environment", False)
gen.add("publish_transforms_updates", bool_t, 5, "Set to True to publish geometry updates of the environment", False)

exit(gen.generate(PACKAGE, PACKAGE, "EnvironmentMonitorDynamicReconfigure"))
